test/Microsoft.AspNetCore.Mvc.Formatters.Json.Test/project.json

```diff
@@ -5,6 +5,7 @@
   "dependencies": {
     "dotnet-test-xunit": "2.2.0-*",
     "Microsoft.AspNetCore.Http": "1.2.0-*",
+    "Microsoft.AspNetCore.JsonPatch": "1.2.0-*",
```

Comment:
This addition shouldn't be necessary. The project already contains tests of the `JsonPatchInputFormatter`.